rules/linux/execution_shell_openssl_client_or_server.toml

```diff
@@ -0,0 +1,95 @@
+[metadata]
+creation_date = "2024/07/30"
+integration = ["endpoint"]
+maturity = "production"
+updated_date = "2024/07/30"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies when the openssl client or server is used to establish a connection. Attackers may use openssl to
+establish a secure connection to a remote server or to create a secure server to receive connections. This activity
+may be used to exfiltrate data or establish a command and control channel.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "endgame-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Openssl Client or Server Activity"
+references = ["https://gtfobins.github.io/gtfobins/openssl/"]
+risk_score = 47
+rule_id = "ad5a3757-c872-4719-8c72-12d3f08db655"
+setup = """## Setup
+
+This rule requires data coming in from Elastic Defend.
+
+### Elastic Defend Integration Setup
+Elastic Defend is integrated into the Elastic Agent using Fleet. Upon configuration, the integration allows the Elastic Agent to monitor events on your host and send data to the Elastic Security app.
+
+#### Prerequisite Requirements:
+- Fleet is required for Elastic Defend.
+- To configure Fleet Server refer to the [documentation](https://www.elastic.co/guide/en/fleet/current/fleet-server.html).
+
+#### The following steps should be executed in order to add the Elastic Defend integration on a Linux System:
+- Go to the Kibana home page and click "Add integrations".
+- In the query bar, search for "Elastic Defend" and select the integration to see more details about it.
+- Click "Add Elastic Defend".
+- Configure the integration name and optionally add a description.
+- Select the type of environment you want to protect, either "Traditional Endpoints" or "Cloud Workloads".
+- Select a configuration preset. Each preset comes with different default settings for Elastic Agent, you can further customize these later by configuring the Elastic Defend integration policy. [Helper guide](https://www.elastic.co/guide/en/security/current/configure-endpoint-integration-policy.html).
+- We suggest selecting "Complete EDR (Endpoint Detection and Response)" as a configuration setting, that provides "All events; all preventions"
+- Enter a name for the agent policy in "New agent policy name". If other agent policies already exist, you can click the "Existing hosts" tab and select an existing policy instead.
+For more details on Elastic Agent configuration settings, refer to the [helper guide](https://www.elastic.co/guide/en/fleet/8.10/agent-policy.html).
+- Click "Save and Continue".
+- To complete the integration, select "Add Elastic Agent to your hosts" and continue to the next section to install the Elastic Agent on your hosts.
+For more details on Elastic Defend refer to the [helper guide](https://www.elastic.co/guide/en/security/current/install-endpoint.html).
+"""
+severity = "medium"
```

Comment:
> As openssl is also used for benign activity, this rule is intended to be tuned by the user

Should we push as low or BBR then?

Reply:
+1

Reply:
I don't think so, because it is still something that we would like to alert on in my opinion. It's not something as low as BBR. I will bump down severity to low however. 

Additionally, only 1 hit in telemetry last 30d is not that bad.